internal/app/machined/pkg/system/services/apid.go

```diff
@@ -170,6 +170,7 @@ func (o *APID) Runner(r runtime.Runtime) (runner.Runner, error) {
 	// Set the mounts.
 	mounts := []specs.Mount{
 		{Type: "bind", Destination: "/etc/ssl", Source: "/etc/ssl", Options: []string{"bind", "ro"}},
+		{Type: "bind", Destination: "/usr/etc", Source: "/usr/etc", Options: []string{"bind", "ro"}},
```

Comment:
I would rather bind mount a specific path (could be conditional) in order not to expose anything else in `/usr/etc` accidentally